PopPUNK/network.py

```diff
@@ -1315,5 +1444,38 @@ def save_network(G, prefix = None, suffix = None, use_gpu = False):
         G.to_pandas_edgelist().to_csv(file_name + '.csv.gz',
                 compression='gzip', index = False)
     else:
-        G.save(file_name + '.gt',
-                fmt = 'gt')
+        if use_graphml:
+            G.save(file_name + '.graphml',
+                    fmt = 'graphml')
+        else:
+            G.save(file_name + '.gt',
+                    fmt = 'gt')
+
+def cugraph_to_graph_tool(G, rlist = None):
```

Comment:
Does `rlist = None` work? Don't we always need rlist provided?

Reply:
You're right, fixed in 5941a3f.